src/components/SearchPageComponents/SidePanel/SidePanel.js

```diff
@@ -0,0 +1,91 @@
+import { Chip, Stack, Box, Typography } from "@mui/material";
+import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
+const SidePanel = ({ searchText }) => {
+  const [list, setList] = useState([]);
+  const [listAdv, setListAdv] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await axios.get(
+          "https://run.mocky.io/v3/41870d6a-bc1f-41f3-a851-67ec27058c03"
+        );
+        setList(res.data);
+      } catch (error) {}
+    };
+    fetchData();
+  }, []);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await axios.get(
+          "https://run.mocky.io/v3/b5ff322d-dabb-409e-9318-0e44742f18fb"
+        );
+        setListAdv(res.data);
+      } catch (error) {}
+    };
+    fetchData();
+  }, []);
+  return (
+    <Stack paddingX={2}>
+      <Typography
+        borderLeft="5px solid "
+        borderColor="secondary.main"
+        paddingLeft={1}
+        fontSize="32px"
+        color="#000"
+      >
+        More results
+      </Typography>
+      <Typography color="#757575" variant="body1" paddingTop={1}>
+        Search "{searchText}" within...
```

Comment:
make sure that searchText will appear , because you're using " instead of ` and ${ }